app/src/main/java/com/main/mealplanner/dao/IRecipeDAO.kt

```diff
@@ -1,6 +1,9 @@
 package com.main.mealplanner.dao
 
-import androidx.room.*
```

Comment:
There are alot of things in room, shouldn't imported everything - just what's needed

Reply:
Performance wise, it won't matter.  But for documentation purposes, yes, it's a good idea to have specific imports and not import *.